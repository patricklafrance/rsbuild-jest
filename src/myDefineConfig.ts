import { defineConfig } from "@rsbuild/core";

export function myDefineConfig() {
    return defineConfig({});
}
