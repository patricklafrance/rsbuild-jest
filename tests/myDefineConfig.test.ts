import { myDefineConfig } from "../src/myDefineConfig";

test("my test", () => {
    const config = myDefineConfig();

    expect(config).toMatchSnapshot();
});
