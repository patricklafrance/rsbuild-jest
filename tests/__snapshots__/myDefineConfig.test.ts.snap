// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my test 1`] = `{}`;
